### PR TITLE
drop ruby sass requirement

### PR DESCRIPTION
**Pull request by bolandrm**
- drop ruby sass requirement
- drop support for sprockets 2

TODO

- [x] fix segfaulting test
- [x] bump to version 2.0.0

### CODE REVIEW

**Comment by bolandrm** [comment]
segfault appears to be an issue with sassc-ruby 2.0.0, as 1.12.1 is working

File: lib/sassc/rails/template.rb [hunk 1]
```diff
@@ -1,73 +1,46 @@
...
-        css = Sprockets::Utils.module_include(::SassC::Script::Functions, @functions) do
-          engine.render
-        end
-
-        context.metadata.merge(data: css)
+  class SassTemplate < Sprockets::SassProcessor
+    def initialize(options = {}, &block)
+      @cache_version = options[:cache_version]
+      @cache_key = "#{self.class.name}:#{VERSION}:#{SassC::VERSION}:#{@cache_version}".freeze
+      #@importer_class = options[:importer] || Sass::Importers::Filesystem
```
Comment (jlw) [inline lib/sassc/rails/template.rb]: This commented line should be removed.

**Comment by deivid-rodriguez** [comment]
For the record, I just tried this branch with [activeadmin](https://github.com/activeadmin/activeadmin/pull/5504) and all tests passed :muscle: :+1:.

**Comment by bolandrm** [comment]
@deivid-rodriguez cool, thanks for testing!  I'll do a release soon

**Comment by jesperronn** [comment]
Whats preventing this PR from being merged?

**Comment by jesperronn** [comment]
Any news on when you'd like to merge/release this, @bolandrm ?

**Comment by bolandrm** [comment]
Sorry, i've been extremely busy.  I'll try to do this tomorrow

**Comment by jesperronn** [comment]
Awesome 👏 I really appreciate your efforts

File: lib/sassc/rails/template.rb (hunk 1 above)
Comment (PikachuEXE) [reply lib/sassc/rails/template.rb]: Still not removed :P

**Pull request merged**